message.js

```diff
@@ -1791,12 +1820,15 @@ jspb.Message.cloneMessage = function(msg) {
  * @param {MESSAGE} toMessage Message which will receive a copy of fromMessage
  *     as its contents.
  * @template MESSAGE
+ * @export
  */
 jspb.Message.copyInto = function(fromMessage, toMessage) {
   goog.asserts.assertInstanceof(fromMessage, jspb.Message);
   goog.asserts.assertInstanceof(toMessage, jspb.Message);
-  goog.asserts.assert(fromMessage.constructor == toMessage.constructor,
-      'Copy source and target message should have the same type.');
+
```

Comment:
im guessing this was added because asserts are now getting eliminated in the bundle and now the tests are failing?

was that intentional or just a side effect of advance optimization

i don't think we want to eliminate asserts, at least not as part of this PR, we should separately consider that

Reply:
This was a side effect of advanced optimization.

Reply:
The same condition is being checked with different format:
```js
  if(fromMessage.constructor !== toMessage.constructor)
      throw new Error('Copy source and target message should have the same type.');
```

Reply:
Ok, sync'd with @dibenede on this issue and we decided that we can proceed with this PR and will just need to close out https://github.com/protocolbuffers/protobuf-javascript/issues/136 before releasing